openbas-api/src/main/java/io/openbas/rest/health_check/HealthCheckApi.java

```diff
@@ -0,0 +1,65 @@
+package io.openbas.rest.health_check;
+
+import io.openbas.rest.helper.RestBehavior;
+import io.openbas.service.HealthCheckService;
+import io.openbas.service.exception.HealthCheckFailureException;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+public class HealthCheckApi extends RestBehavior {
+
+  public static final String HEALTH_CHECK_URI = "/api/health";
+
+  private HealthCheckService healthCheckService;
+
+  private String healthCheckKey;
+
+  @Autowired
+  public void setHealthCheckService(HealthCheckService healthCheckService) {
+    this.healthCheckService = healthCheckService;
+  }
+
+  @Autowired
+  public void setHealthCheckKey(
+      @Value("${openbas.healthcheck.key:#{null}}") String healthCheckKey) {
+    this.healthCheckKey = healthCheckKey;
+  }
+
+  @GetMapping(HEALTH_CHECK_URI)
+  @Operation(
+      summary = "Run an healthcheck ",
+      description = "Tries to connect to dependencies (DB/Minio/RabbitMQ)")
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "Service is healthy"),
+        @ApiResponse(responseCode = "503", description = "Service is not running properly")
+      })
+  public ResponseEntity<?> healthCheck(
+      @RequestParam("health_access_key") String requestHealthAccessKey) {
+    if (StringUtils.isEmpty(requestHealthAccessKey)
```

Comment:
nitpick: `StringUtils.isEmpty` returns false even when string only consists of whitespace. Consider using `isBlank` to avoid an accidental misconfiguration with spaces (e.g. typo in config) https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty(java.lang.CharSequence)

Reply:
yes that's better, I will update it in the next commit